test/e2e/framework/util.go

```diff
@@ -3900,6 +3900,23 @@ func WaitForControllerManagerUp() error {
 	return fmt.Errorf("waiting for controller-manager timed out")
 }
 
+// Returns cluster size (number of ready Nodes excluding Master Node).
+func ClusterSize(c clientset.Interface) (int, error) {
```

Comment:
It shouldn't be called ClusterSize. It should be something like NumberOfSchedulableNodes or sth like that.

Reply:
It's called like this because of the function immediately below (WaitForClusterSize). I can rename this function, but then for consistency's sake I should rename the other one two. Should I do this?

Reply:
Yeah - we should rename both.

Reply:
Done. PTAL.